docs/conf/portland/2018/sponsors/prospectus.rst

```diff
@@ -163,52 +174,41 @@ The **Keystone** package costs **$14,000**.
 Other Sponsorship Opportunities
 ===============================
 
-The following à la carte offerings are available either independently or
+The following a la carte offerings are available either independently or
 combined with one of the previous packages to increase visibility at the event.
 
-Financial Assistance
---------------------
+Job Fair
+--------
 
-Write the Docs is always trying to make our community more diverse and
-inclusive. Talk to us about sponsoring travel and/or registration expenses for
-an attendee who wouldn’t otherwise be able to attend.
+Our job fair is a wonderful place to connect with the talented Write the Docs attendees.
+We'll have over 400 people in attendance for our conference,
+and many of those people will come to our job fair to look at possible new directions in the industry.
 
-Lanyard
--------
+**$1,000** (a la carte price, also included or discounted with other sponsorship packages)
 
-Provide your branded lanyards for our name tags.
+- **Logistics**: The Job Fair will be Tuesday morning in Lola's Room. It will run from 9am-11:30pm. Setup at 8am.
```

Comment:
do we need to specify *they* setup?

Reply:
+1, they need to set their stuff up, we just provide the tables